### DISCUSSION TITLE
check if the runner is idle or running something

### DISCUSSION THREAD

**Discussion by Puneeth-n** [actions/actions-runner-controller#554] [Questions] [answered]
Is it possible to find from kubectl if the runners are idle or running something? Instead of going to the GitHub Actions settings?

**Comment by mumoshu** [comment, marked as answer]
@Puneeth-n Have you tried `kubectl get runner`? `STATUS` should be `Running` only after your runner is running.

**Reply by Puneeth-n** [reply]
does it mean that if it is not processing any builds, it says `idle`

**Reply by fieldsfarmer** [reply]
Yes
